### ISSUE TITLE
Nothing happening

### ISSUE THREAD

**Issue by skatermem** [freshplanet/ANE-Google-Play-Game-Services#14]
I use flash cs6 and adobe air 13. I have all the required code in my project and I have google games support set up in my developers account. I have 
AirGooglePlayGames.getInstance().addEventListener(AirGooglePlayGamesEvent.ON_SIGN_IN_SUCCESS, onSignInSuccess);
AirGooglePlayGames.getInstance().addEventListener(AirGooglePlayGamesEvent.ON_SIGN_OUT_SUCCESS, onSignOutSuccess);
AirGooglePlayGames.getInstance().addEventListener(AirGooglePlayGamesEvent.ON_SIGN_IN_FAIL, onSignInFail);
AirGooglePlayGames.getInstance().startAtLaunch();
I have tried AirGooglePlayGames.getInstance().signIn();
I have this in my xml file: (***\* is just representing my developer id)
meta-data android:name="com.google.android.gms.games.APP_ID" android:value="\ *********"
meta-data android:name="com.google.android.gms.version
android:value="@integer/google_play_services_version
activity android:name="com.freshplanet.googleplaygames.SignInActivity
android:theme="@android:style/Theme.Translucent.NoTitleBar.Fullscreen
 I don't get an error but nothing happens in my game, I have created a sign in button, nothing happens, in ddms Activity Manager starts and the text says Start u0 followed by my games name but nothing happens on-screen.  I also made a sign out button, and the strange thing is when i click on it, in ddms i see it tracing out sign out successful and gamehelper says signout: state was already disconnected,ignoring but i never signed in. Also my game does not show up in Google+ connected apps. I have tried everything. Can anyone make sense of this?

**Comment by kyoshyro86** [comment]
I have same problem. My game is linked to the API console project called 'clashofcrush'  and my package name is air.air.ClashofCrush. and that crash when i run in device. Please help me

**Comment by gotohell32434** [comment]
I never got it to work. I never got a single ANE to work, in fact. They all fail.

**Comment by ShockWave2048** [comment]
Is it exists working free ANE for Google Games Services?

**Comment by Laur3nt1u** [comment]
my fork works

**Comment by ShockWave2048** [comment]
will try, tnks!

**Comment by Laur3nt1u** [comment]
you'll need [Google-Play-Game-Services](https://github.com/Laur3nt1u/ANE-Google-Play-Game-Services) and [Google-Play-Services](https://github.com/Laur3nt1u/ANE-Google-Play-Services)

**Comment by ShockWave2048** [comment]
Both needed?

**Comment by Laur3nt1u** [comment]
yes, but you'll have problems if you use another ane with google play services. What ane's you have in your project?

**Comment by ShockWave2048** [comment]
no ane with gps, another only freeAneInAppShare.ane (may removed)

**Comment by ShockWave2048** [comment]
Big thanks Laur3nt1u!!!!
It's working now!!!
